projects/igniteui-angular/src/lib/time-picker/time-picker.component.html

```diff
@@ -1,95 +1,80 @@
-<ng-template #labelTemplate>
-    <ng-content select="[igxLabel]"></ng-content>
-</ng-template>
+<!-- [preventSpinOnWheel]="!this.isDropdown" -->
+<igx-input-group [suppressInputAutofocus]="this.isDropdown" (click)="!this.isDropdown && this.open()">
+    <input igxInput [igxDateTimeEditor]="this.inputFormat" type="text" [readonly]="!this.isDropdown"
```

Comment:
This is also missing some component properties I think, `placeholder`  and `spinLoop` at least